### DISCUSSION TITLE
Tailwind/CSS help: Want ul container to show only first 5 results but still allow user to scroll through list to find their correct location

### DISCUSSION THREAD

**Discussion by HolisticDev1992** [tailwindlabs/tailwindcss#10796] [Help] [answered]
![image](https://user-images.githubusercontent.com/115614967/225170777-15ec95a6-531e-40e5-b20c-c740db6b9f82.png)

**Comment by bkpecho** [comment, marked as answer]
You could set a max height on the lists container and set overflow-y to auto

**Reply by HolisticDev1992** [reply]
Thanks!
